vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -174,31 +142,24 @@ export default {
           event: "tap_header_cta",
         },
       ],
-      hamburgerNavbars: [
-        {
-          name: "Services",
-          url: "/services",
-          event: "tap_header_services",
-        },
-        {
-          name: "Resources",
-          url: "/resources",
-          target: "_self",
-          event: "tap_header_resources",
-        },
-      ],
     };
   },
   components: {
     FontAwesomeIcon,
   },
   mounted() {
-    if (Config.SHOW_SERVICES_PAGE && Config.SHOW_RESOURCES_PAGE) {
+    if (Config.SHOW_SERVICES_PAGE) {
```

Comment:
Move `let's talk` at last in mobiles, also remove space between menus and button on desktop